src/Lesson1/HomeWorkApp.java

```diff
@@ -0,0 +1,45 @@
+package Lesson1;
+
+public class HomeWorkApp {
+
+    public static void main(String[] args) {
+        printThreeWords();
+        checkSumSign();
+        printColor();
+        compareNumbers();
+    }
+    public static void printThreeWords() {
+        System.out.println("Orange");
+        System.out.println("Banana");
+        System.out.println("Apple");
+    }
+
+    public static void checkSumSign() {
+        int a=7, b=15;
+        if ((a+b)>=0) {
```

Comment:
Используйте ctrl+alt+L для автоформатирования кода.
```suggestion
        if ((a+b) >= 0) {
```